src/choreo/ghost/fd_ghost.c

```diff
@@ -448,34 +489,37 @@ fd_ghost_publish( fd_ghost_t * ghost, ulong slot ) {
 }
 
 fd_ghost_ele_t const *
-fd_ghost_gca( fd_ghost_t const * ghost, ulong slot1, ulong slot2 ) {
+fd_ghost_gca( fd_ghost_t const * ghost, fd_hash_t const * bid1, fd_hash_t const * bid2 ) {
```

Comment:
think it's easier to avoid block_id terminology in the code itself

```suggestion
fd_ghost_gca( fd_ghost_t const * ghost, fd_hash_t const * hash1, fd_hash_t const * hash2 ) {
```